sdks/python/tox.ini

```diff
@@ -48,6 +48,11 @@ deps =
 whitelist_externals=find
 commands =
   python --version
+  # Clean up all previous cython generated files.
+  - find apache_beam -type f -name '*.c' -delete
```

Comment:
What does the dash signify? I think I've gotten errors "bash cannot find command '-'" when running code like this.